R/conversion.R

```diff
@@ -160,7 +160,29 @@ py_to_r.pandas.core.series.Series <- function(x) {
   py_to_r(x$as_matrix())
 }
 
+#' @export
+summary.pandas.core.series.Series <- function(object, ...) {
+  if (py_is_null_xptr(object) || !py_available())
+    str(object)
```

Comment:
I wasn't sure what the best default summary this should be but I believe this was the default prior to this PR (see the example in the PR description).

Reply:
This should now print this when the xptr is NUlL:

```r
<pointer: 0x0>
```

Can you confirm that is what you see as well as test the other methods with null xptrs for both series and data frame? (extra important to test these as if we don't get them right they can cause crashing and/or binding to the wrong version of Python).

Reply:
Yes, I confirmed by using `saveRDS` and `readRDS` on both pandas data frame and series. They yield the expected results:
```
> pdf
<pointer: 0x0>
> dim(pdf)
NULL
> length(pdf)
[1] 0
> summary(pdf)
<pointer: 0x0>
```

Reply:
okay, we are good to merge then once @kevinushey has a look